src/main/java/TaskList.java

```diff
@@ -0,0 +1,18 @@
+import java.util.ArrayList;
+
+public class TaskList {
+    ArrayList<Task> arrTask = new ArrayList<Task>();
```

Comment:
Perhaps the instantiation here should be moved under the default constructor instead? Since the field declaration does not need an instantiation.